src/test/regress/sql/multi_task_assignment_policy.sql

```diff
@@ -261,5 +261,43 @@ SELECT count(DISTINCT value) FROM explain_outputs;
 RESET citus.task_assignment_policy;
 RESET client_min_messages;
 
+--  make sure we set the right anchor shardid for task
+-- citus/issues/2749 , anchor_shardId should be assigned as dis_table'shardid 
+SET citus.shard_replication_factor TO 1;
+SET citus.replication_model TO streaming;
+CREATE TABLE reference_table (id bigint, value bigint);
+SELECT create_reference_table('reference_table');
+
+CREATE TABLE dis_table (
```

Comment:
We typically try to create/drop as less distributed tables as possible in the regression tests, mostly to prevent the regression test times keep increasing.

So, instead of creating new tables, could you please use the existing tables: instead of `dist_table` could use `task_assignment_nonreplicated_hash ` and instead of `reference_table` could use `task_assignment_reference_table `. 

Thanks!